tests/test_utils/run_server.py

```diff
@@ -8,11 +8,11 @@
 from .layers import name_to_block, name_to_input
 
 
-def make_dummy_server(host='0.0.0.0', port=None, num_experts=1, expert_cls='ffn', hidden_dim=1024, num_handlers=None,
-                      expert_prefix='expert', expert_offset=0, max_batch_size=16384, device=None, no_optimizer=False,
-                      no_dht=False, initial_peers=(), dht_port=None, root_port=None, verbose=True, start=False,
-                      UID_DELIMETER=hivemind.DHTNode.UID_DELIMETER, **kwargs) -> hivemind.Server:
-    """ A context manager that creates server in a background thread, awaits .ready on entry and shutdowns on exit """
+def make_dummy_server(interface='0.0.0.0', port=None, num_experts=1, expert_cls='ffn', hidden_dim=1024,
+                      num_handlers=None, expert_prefix='expert', expert_offset=0, max_batch_size=16384, device=None,
+                      no_optimizer=False, no_dht=False, initial_peers=(), dht_port=None, root_port=None, verbose=True,
+                      UID_DELIMETER=hivemind.DHTNode.UID_DELIMETER, start=False, **kwargs) -> hivemind.Server:
+    """ Instantiate a server with several identical experts. See argparse comments below for details """
```

Comment:
This is a function which is called separately from argparse (in fact, run_server.py is never called in tests). If we're trying to use this function elsewhere (think of IDE/Jupyter), the docstring with references to argparse is not going to be very useful. I suggest we move/copy the description of parameters here instead of argparse options